weekly-scan/template.yaml

```diff
@@ -0,0 +1,145 @@
+apiVersion: "v1"
+kind: "Template"
+metadata:
+  name: "weekly-scan-job-template"
+objects:
+  - apiVersion: "v1"
+    kind: "BuildConfig"
+    metadata:
+      name: ${PIPELINE_NAME}
+    spec:
+      strategy:
+        type: "JenkinsPipeline"
+        jenkinsPipelineStrategy:
+          jenkinsfile: |
+            properties([
+                pipelineTriggers([
+                    cron('0 23 * * 6'),
```

Comment:
That's Friday 11pm, I was thinking if we should have weekly scans on Thursday 11pm, so we could triage/inspect it on Friday (and possibly resolve any issues etc) 

@mohammedzee1000 @dharmit @bamachrn WDYT ?

Reply:
I will fix this once we have a vote on the best time to run weekyl scan. Right now it is Saturday 11 pm, not Friday.